build/cmd.sh

```diff
@@ -402,6 +402,19 @@ function build_image_internal {
 function install_vendor_internal {
     if [ ! -d vendor ]; then
         glide install --strip-vendor
+
+	# handle a few cases
+	pushd /go/src/github.com/Mirantis/virtlet/vendor/github.com/coreos/
+	mkdir v22
```

Comment:
why v22 and v5? why change the location at all?

Reply:
the cgroup package in containerd uses this path in its import in cgroup.go file. the godep tool apparently put the clone to the specific path however the old glide tool doesn't have such flexibility. hence this workaround to put them to the right desired path in the build env.